server/src/api/modules/efficiency/configs/ehp/f2p_ironman.ehp.ts

```diff
@@ -46,42 +46,34 @@ export default [
       },
       {
         startExp: 13_034_431,
-        rate: 50_100,
-        description: 'Hill Giants & Ogresses'
+        rate: 51_200,
+        description: 'Hill Giants'
       }
     ],
     bonuses: [
       {
         originSkill: Skill.ATTACK,
         bonusSkill: Skill.PRAYER,
-        startExp: 4_825,
+        startExp: 37_224,
         endExp: 13_034_431,
         end: true,
-        ratio: 0.125
+        ratio: 0.129
```

Comment:
Is the 0.125 prayer bonus being removed from attack `4,825 -> 37,224` xp? Your sheet still shows that bonus.

Reply:
Good catch ty